strongbox-security/strongbox-user-management/src/main/java/org/carlspring/strongbox/authorization/AuthorizationConfigFileManager.java

```diff
@@ -45,4 +55,58 @@ public ConfigurationResourceResolver getConfigurationResourceResolver()
         return configurationResourceResolver;
     }
 
+    @Override
+    public synchronized void store(AuthorizationConfigDto configuration) throws IOException
+    {
+        Map<SystemRole, String> changed = getChangedRestrictedRoles(configuration);
+        if (changed.size() > 0)
+        {
+            String message = String.join("; ", changed.values());
```

Comment:
I think there should be enough something like "System roles %s  cannot be changed." as error message.

Reply:
done